aim/api/resource.py

```diff
@@ -1496,3 +1496,285 @@ def __init__(self, **kwargs):
+    and src_path of the VSPAN session.
+    """
+
+    identity_attributes = t.identity(
+        ('vsg_name', t.name),
+        ('vs_name', t.name),
+        ('src_path', t.string()))
+    other_attributes = t.other(
+        ('monitored', t.bool))
+
+    _aci_mo_name = 'spanRsSrcToVPort'
+    _tree_parent = SpanVsource
+
+    def __init__(self, **kwargs):
+        super(SpanSrcVport, self).__init__({'monitored': False},
+                                           **kwargs)
+
+
+class SpanVdestGroup(AciResourceBase):
+    """Resource representing a ERSPAN VDest Group in ACI.
+
+    Identity attributes are RNs for ERSPAN VDest Group.
+    """
+
+    identity_attributes = t.identity(
+        ('name', t.name))
+    other_attributes = t.other(
+        ('monitored', t.bool),
+        ('display_name', t.name))
+
+    _aci_mo_name = 'spanVDestGrp'
+    _tree_parent = Infra
+
+    def __init__(self, **kwargs):
+        super(SpanVdestGroup, self).__init__({'monitored': False},
+                                             **kwargs)
+
+
+class SpanVdest(AciResourceBase):
+    """Resource representing a ERSPAN VDest in ACI.
+
+    Identity attributes are RNs for ERSPAN VDest.
+    """
+
+    identity_attributes = t.identity(
+        ('vdg_name', t.name),
+        ('name', t.name))
+    other_attributes = t.other(
+        ('monitored', t.bool),
+        ('display_name', t.name))
+
+    _aci_mo_name = 'spanVDest'
+    _tree_parent = SpanVdestGroup
+
+    def __init__(self, **kwargs):
+        super(SpanVdest, self).__init__({'monitored': False},
+                                        **kwargs)
+
+
+class SpanVepgSummary(AciResourceBase):
+    """Resource representing a ERSPAN Destination VEpg Summary in ACI.
+
+    Identity attributes are RNs for ERSPAN Destination VEpg Summary.
+    """
+
+    identity_attributes = t.identity(
+        ('vdg_name', t.name),
+        ('vd_name', t.name))
+    other_attributes = t.other(
+        ('monitored', t.bool),
+        ('display_name', t.name),
+        ('dst_ip', t.string()),
+        ('flow_id', t.integer),
+        ('ttl', t.integer),
+        ('mtu', t.integer))
+
+    _aci_mo_name = 'spanVEpgSummary'
+    _tree_parent = SpanVdest
+
+    def __init__(self, **kwargs):
+        super(SpanVepgSummary, self).__init__({'monitored': False,
+                                               'dst_ip': '',
+                                               'flow_id': 1,
+                                               'ttl': 64,
+                                               'mtu': 1518},
+                                              **kwargs)
+
+
+class InfraAccBundleGroup(AciResourceBase):
+    """Resource representing bundle interface group in ACI.
+
+    Identity attribute is bundled ports group name.
+    """
+
+    identity_attributes = t.identity(
+        ('name', t.name))
+    other_attributes = t.other(
+        ('monitored', t.bool),
+        ('display_name', t.name))
+
```

Comment:
According to https://pubhub.devnetcloud.com/media/apic-mim-ref-501/docs/MO-infraAccBndlGrp.html, this is missing lagT (Link Aggregation Type).